build.gradle

```diff
@@ -180,6 +180,9 @@ def versions = [
 dependencies {
     compile project(':am-lib')
 
+    annotationProcessor group: 'org.projectlombok', name: 'lombok', version: '1.18.6'
```

Comment:
Why is that needed?

Reply:
because we have moved the FilterResourceModel into the am-lib testing service

Reply:
Good point. I have forgotten about that move. 👍